### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by rene-hermenau**

### CODE REVIEW

File: wp-staging-hooks.php [hunk 1]
```diff
@@ -483,6 +490,7 @@ function wpstg_backup_files_ignore_files_w_extension_bigger_than($default){
   
     /**
      * Backup: Exclude folders from being included in a backup
+     * E.g. This examples excludes the folder wp-content/cache
```
Comment (mauriac) [inline wp-staging-hooks.php]: May be use this example: `WP_CONTENT_DIR . '/uploads/wp-staging/cache`, I think we need to highlight the fact that this filter works with absolute path not relative one!

Comment (rene-hermenau) [inline wp-staging-hooks.php]:
```suggestion
     * E.g. This examples excludes the folder WP_CONTENT_DIR  . '/cache'
```

Comment (rene-hermenau) [reply wp-staging-hooks.php]: Done. Afaik: Folder uploads/wp-staging/ including its cache folder is never included in backup and always excluded. So I've changed the example.

**Comment by rene-hermenau** [comment]
@mauriac Please give another review

File: wp-staging-hooks.php (hunk 1 above)
Comment (mauriac) [reply wp-staging-hooks.php]: > Done. Afaik: Folder uploads/wp-staging/ including its cache folder is never included in backup and always excluded. So I've changed the example.

yes, you are totally right! I meant to change the exemple while I was copying it... thanks!

**Comment by mauriac** [comment]
all good for me!

**Review by mauriac — APPROVED** [review]

**Pull request merged**